contrib/IECoreArnold/src/IECoreArnold/outputDriver/OutputDriver.cpp

```diff
@@ -134,6 +137,12 @@ void driverOpen( AtNode *node, struct AtOutputIterator *iterator, AtBBox2 displa
 			namePrefix = std::string( name ) + ".";
 		}
 
+		const StringData *label = parameters->member< StringData >( "label" );
```

Comment:
I thought we'd agreed on calling this "layerName", since that's what it is from a GafferImage standpoint? It also seems pretty unexpected that the output driver would hardcode a special case specific to one pipeline ("beauty") - can that be dealt with in IERendering?